engine/crash/src/test/test_script_crash.cpp

```diff
@@ -49,7 +49,11 @@ class ScriptCrashTest : public jc_test_base_class
 
         dmResource::NewFactoryParams factory_params;
         m_ResourceFactory = dmResource::NewFactory(&factory_params, ".");
-        m_Context = dmScript::NewContext(m_ConfigFile, m_ResourceFactory, true);
+
+        dmScript::ContextParams script_context_params = {};
+        script_context_params.m_Factory = m_ResourceFactory;
+        script_context_params.m_ConfigFile = m_ConfigFile;
+        m_Context = dmScript::NewContext(script_context_params);
```

Comment:
The signature has slightly changed, but it's not part of dmsdk so no breaking change.